storm-server/src/main/java/org/apache/storm/daemon/nimbus/Nimbus.java

```diff
@@ -1081,6 +1088,14 @@ public Nimbus(Map<String, Object> conf, INimbus inimbus, IStormClusterState stor
             BlobStore blobStore, TopoCache topoCache, ILeaderElector leaderElector, IGroupMappingServiceProvider groupMapper)
         throws Exception {
         this.conf = conf;
+
+        this.metricsStore = null;
```

Comment:
Can we verify that if the metricsStore is null that we don't get any NullPointerExceptions in really bad places.

Reply:
Re-verified that a null MetricStore should be safe.